.github/workflows/build.yml

```diff
@@ -99,7 +99,21 @@ jobs:
             }
           }
           return "FALSE";
-    - uses: actions/checkout@v3
```

Comment:
I'm not sure how I can debug this for releases and pushes with release tags and for develop.

Maybe this can be refactored in a nicer way as it seems like right now most checkouts except develop are done via the SHA and not the ref and therefore the branch will just show HEAD inside of the docker images.

Reply:
Yes - but always prefix them with some clear "test" indicator ;)

Reply:
@Mtze I just tagged the last commit in this PR with `test/action1` but I just realized that this doesn't trigger the push because it's not in the desired format `[0-9]+.[0-9]+.[0-9]+`. I could use `0.0.100` for instance or sth similar for testing?

Reply:
ah - you're correct. Yes - you can do that

Reply:
tried it with `0.0.100` it checksout the tag but then I guess it's not displayed because we always just show the branch which then is just `HEAD` in this case. But this would have to be changed in the code at multiple places one of them is here:
https://github.com/ls1intum/Artemis/blob/develop/src/main/java/de/tum/in/www1/artemis/ArtemisApp.java#LL78

Reply:
So fixing this is a follow-up PR?

Reply:
Yes - the version is displayed in the top bar anyway